cypress/e2e/1-login/login.cy.js

```diff
@@ -0,0 +1,58 @@
+import {apiPath} from "../../../src/js/api";
+
+describe("The social media app", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  })
+
+  it("can login a user and access their profile", () => {
+    cy.intercept('POST', `${apiPath}/social/auth/register`, (req) => {
+      req.reply({
```

Comment:
I think this is the problem. You are forcing a fake reply, instead of using the reply from the server